src/Utilities/SafeMethods.php

```diff
@@ -4,20 +4,30 @@
 
 namespace BeechIt\JsonToCodeClimateSubsetConverter\Utilities;
 
+use BeechIt\JsonToCodeClimateSubsetConverter\Exceptions\FilesystemException;
+use BeechIt\JsonToCodeClimateSubsetConverter\Exceptions\JsonException;
+use BeechIt\JsonToCodeClimateSubsetConverter\Exceptions\StringsException;
 use BeechIt\JsonToCodeClimateSubsetConverter\Interfaces\SafeMethodsInterface;
+use function error_clear_last;
+use function json_last_error;
 
+/**
+ * @codeCoverageIgnore
+ */
 class SafeMethods implements SafeMethodsInterface
 {
     public function json_encode(
         $value,
         int $options = self::JSON_ENCODE_OPTIONS,
         int $depth = self::JSON_ENCODE_DEPTH
     ): string {
-        return \Safe\json_encode(
-            $value,
-            $options,
-            $depth
-        );
+        error_clear_last();
+        $result = \json_encode($value, $options, $depth);
```

Comment:
You can `use` this as well

Reply:
+1 and we can automate that via `php-cs-fixer` https://mlocati.github.io/php-cs-fixer-configurator/#version:3.0|fixer:global_namespace_import